locales/en/moves.json

```diff
@@ -114,5 +114,8 @@
       "content": "Reference number<br><strong>{{moveReference}}</strong>"
     },
     "detail": "The move for <strong><a href=\"{{href}}\">{{name}}</a></strong> to <strong>{{location}}</strong> on <strong>{{date}}</strong> has been scheduled."
+  },
+  "details": {
```

Comment:
Keys in the steps object in locales should match the step name so they're easier to find: `agreement-status`